### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by armellarcier**
Fix Theming docs

### CODE REVIEW

**Review by jacquesbh — CHANGES_REQUESTED** [review]
We are not giving the new path but where to find the files to override.

If you want you can add a link to the documentation of Sylius about theming. It could be better.

**Comment by armellarcier** [comment]
Done @jacquesbh

File: README.md [hunk 1]
```diff
@@ -290,11 +290,18 @@ By default, the form is displayed on `sonata.block.event.sylius.shop.layout.head
...
+The bundle's templates are : 
+- Search results display page (`src/MonsieurBizSearchPlugin/Resources/views/Search/`)
+- Instant search display block (`src/MonsieurBizSearchPlugin/Resources/views/Instant/`)
+- Taxon results display page (`src/MonsieurBizSearchPlugin/Resources/views/Taxon/`)
+- Smaller components (`src/MonsieurBizSearchPlugin/Resources/views/Common/`)
+- JS parameters (`src/MonsieurBizSearchPlugin/Resources/views/js.html.twig`)
+
+Sylius documentation to customize these templates is available [here](https://docs.sylius.com/en/latest/customization/template.html).
+
+E.g. To override the filter's template create `themes/MyTheme/MonsieurBizSyliusSearchPlugin/views/Common/_filter.html.twig`
```
Comment (jacquesbh) [inline README.md:304]: No need of this line I think. It's not the purpose of this readme to explain how to develop on Sylius. :)

**Review by jacquesbh — CHANGES_REQUESTED** [review]
One last thing.
Otherwise it's ok for me.

File: README.md (hunk 1 above)
Comment (armellarcier) [reply README.md]: I knew it ;) done

**Review by jacquesbh — APPROVED** [review]

**Comment by jacquesbh** [comment]
Thank you @armellarcier ! ✏️ 💪

**Pull request merged**